test/commands/environments/copy-layout.test.js

```diff
@@ -45,7 +45,17 @@ describe('environments:copy-layout', () => {
         parsedBody = body;
         return body;
```

Comment:
I would have called it payload, wdyt ?

Reply:
Please feel free to do it (your PR is small)